pyngrok/process.py

```diff
@@ -372,7 +373,12 @@ def _start_process(pyngrok_config):
         logger.info("Starting ngrok in region: {}".format(pyngrok_config.region))
         start.append("--region={}".format(pyngrok_config.region))
 
-    proc = subprocess.Popen(start, stdout=subprocess.PIPE, universal_newlines=True)
+    popen_kwargs = {'stdout': subprocess.PIPE,  'universal_newlines' : True}
+    if sys.version_info.major >= 3:
+        popen_kwargs.update(start_new_session=pyngrok_config.start_new_session)
+    elif pyngrok_config.start_new_session:
+        logger.warning("Ignoring start_new_session=True, which requires Python 3")
```

Comment:
This seems a little noisy here, I'd probably throw such a warning when the config is initialized instead. Also let's us get rid of the Python version check code here (move it to the config initialization as well) and here we simply check if the config var is set to True, and if so append it to the kwargs as you're doing.

Reply:
I think it's better here, because it is possible to call the default config constructor, then assign `True` to the `start_new_session` variable.  I did this in my test case, precisely because I was worried about it.